libc/src/__support/common.h

```diff
@@ -21,9 +21,24 @@
 #define LLVM_LIBC_FUNCTION_ATTR
 #endif
 
+// Allow each function `func` can have extra attributes specified by defining:
+// `LLVM_LIBC_FUNCTION_ATTR_func` macro, which should always start with
+// "LLVM_LIBC_EMPTY,"
+//
+// For example:
+// #define LLVM_LIBC_FUNCTION_ATTR_memcpy LLVM_LIBC_EMPTY, __attribute__((weak))
```

Comment:
Should this be `LLVM_LIBC_EMPTY,__attribute__((weak))` (w/o space)?

If not, use `"LLVM_LIBC_EMPTY, "` on line 26 above.

Reply:
Updated line 26.  Passing the macro from `EXPAND_THEN_SECOND` to `SECOND`  will ignore the spaces there, give some extra spaces should make it better for readability.